ledger-tool/src/main.rs

```diff
@@ -3245,6 +3249,10 @@ fn main() {
                                     warn!("Minimized snapshot range crosses epoch boundary ({} to {}). Bank hashes after {} will not match replays from a full snapshot",
                                         starting_epoch, ending_epoch, bank.epoch_schedule().get_last_slot_in_epoch(starting_epoch));
                                 }
+
+                                if minimize_snapshot_possibly_incomplete {
+                                    warn!("Minimized snapshot may be incomplete due to missing accounts from CPI'd address lookup table extensions. This may lead to replay failures");
```

Comment:
`This may lead to replay failures` => `This may lead to mismatched bank hashes while replaying`. Missing actually accessed accounts in minimized snapshot won't cause hard errors as indicated by `replay failures`. it's more subtle and scary bank hash mismatches out of sudden. that's because those missed accounts are just filled `Account::default()` and on-chain programs just start to behave differently. (most likely just erring..)